docs/guides/getting-started/setup/vite.mdx

```diff
@@ -127,6 +133,14 @@ Now that we have scaffolded our frontend and initialized the Rust project the cr
   },
 ```
 
+Be sure that tauri is accessible to your package manager of choice. For example if you are using npm, the `package.json` file should include tauri as a script:
+```json title=package.json
+{
+  "scripts": {
+    "tauri": "tauri"
+  },
+```
+
```

Comment:
Writing this warning here would be too late since `tauri init` needs it too. I think it should be part of the `Install Tauri CLI` tabber instead.

Reply:
Agreed. I just wanted to make sure the user didn't think they needed to install tauri as a dependency or dev-dependency.